data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                    "x_is_acknowledged": false,
+                    "severity": 90,
+                    "time_observed": "2023-12-14T09:14:39.802Z",
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "Bad IP reputation (new node)",
+                    "x_cause": "An IP address which is known to be malicious has appeared in the network. Likely some kind of malware is attempting to perform malicious activity",
+                    "ioc_refs": [
+                        "8"
+                    ],
+                    "x_solution": "Get in touch with the local site to make sure that any kind of malware is removed from the involved Assets.",
+                    "ttp_tagging_refs": [
+                        "9",
+                        "10"
+                    ],
+                    "description": "New IP node 109.205.213.34 has appeared, that is known to have bad reputation [RDP bruteforce]",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "x_threat_name": "RDP bruteforce",
+                    "finding_type": "threat",
+                    "start": "2023-12-14T09:14:39.802Z",
+                    "x_sensor_host": "EC2AMAZ-XXXXX",
+                    "x_sensor_interface": "remote_capture",
+                    "x_alert_type_id": "VI:NEW-NODE:MALICIOUS-IP"
+                },
+                "1": {
+                    "type": "ipv4-addr",
+                    "value": "2.2.2.2",
+                    "resolves_to_refs": [
+                        "4"
+                    ],
+                    "x_nozomi_info_ref": "6"
+                },
+                "2": {
+                    "type": "network-traffic",
+                    "dst_ref": "1",
+                    "src_ref": "3",
+                    "dst_port": 3389,
+                    "src_port": 43938,
+                    "protocols": [
+                        "tcp",
+                        "tcp"
+                    ]
+                },
+                "3": {
+                    "type": "ipv4-addr",
+                    "value": "3.3.3.3",
+                    "resolves_to_refs": [
+                        "5"
+                    ],
+                    "x_nozomi_info_ref": "7"
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "12:xx:de:xx:84:e5",
+                    "x_nozomi_info_ref": "6"
+                },
+                "5": {
+                    "type": "mac-addr",
+                    "value": "12:xx:9a:xx:a5:96",
+                    "x_nozomi_info_ref": "7"
+                },
+                "6": {
+                    "type": "x-nozomi-info",
+                    "zone": "1.1.1.1/21",
+                    "roles": "other",
+                    "label": "EC2AMAZ-XXXXX",
```

Comment:
This seems to be a partially obscured real hostname. When obscuring values, it's best to remove any references to the real data and use a defined replacement. For example "Hostname" or "Host" might work better.